README.md

```diff
@@ -109,6 +109,7 @@ type GoCloak interface {
 	LoginOtp(ctx context.Context, clientID, clientSecret, realm, username, password, totp string) (*JWT, error)
 	Logout(ctx context.Context, clientID, clientSecret, realm, refreshToken string) error
 	LogoutPublicClient(ctx context.Context, clientID, realm, accessToken, refreshToken string) error
+	LogoutAllSessions(realm, userID, accessToken string) error
```

Comment:
please copy the function from the https://github.com/Nerzal/gocloak/pull/186/files#diff-85bb9be27874ff6ac3ca4b4cc9585477R30

Reply:
Good that u have an eye for the details <3